docs/index.html

```diff
@@ -33,10 +33,12 @@ <h1 class="heading-xlarge u-space-b15">MOT history API</h1>
 
     <div class="grid-row u-space-t30">
         <div class="column-two-thirds">
-            <p>It's no longer possible to register for this version of the MOT history API.</p>
+            <p>A new version of the MOT history API is available. <a
+                    href="https://documentation.history.mot.api.gov.uk/" target="_blank">You can register to use the new API now.</a></p>
```

Comment:
"You can" should not be part of the link text according to content in the ticket